code/platforms/win/app.py

```diff
@@ -1,11 +1,47 @@
 # defines the default app actions for windows
 
-from talon import Context, actions
+import win32gui
+from talon import Context, actions, ui
 ctx = Context()
 ctx.matches = r"""
 os: windows
 """
 
+# adapted this from code posted by @PeterLinder
+# return list of top level windows
```

Comment:
what are top-level windows? could you add a comment here explaining the need for this?

Reply:
Don't rightly know, TBH...except that we want to exclude non-top-level windows while switching with this command. The m$ docs could be clearer on this point.

Added a (snarky) comment.